walkthroughs/week-2-server/examples/server-stats/src/main/webapp/script.js

```diff
@@ -39,3 +39,11 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+fetch('/my-data-url')  // sends a request to /my-data-url
+.then(response => response.json()) // parses the response as JSON
+.then((myObject) => { // now we can reference the fields in myObject!
+  console.log(myObject.x);
+  console.log(myObject.y);
+  console.log(myObject.z);
+});
```

Comment:
Let's avoid making changes to the example code.

Reply:
Still not done?

Reply:
I believe I made the change. I don't see this in my example code locally but when I try to push this happens:
`$ git add .
 $ git commit -m "Update"
On branch P3_Dev
Changes not staged for commit:
      modified:   ../walkthroughs/week-2-server/examples/server-stats/src/main/webapp/script.js
no changes added to commit`

Reply:
Fixed!! Ah, so after some research (random testing), I tried `git add -A` instead of my goto `git add .`.
Is there a reason one worked over the other. Should I not use '.' (dot), but rather '-A'?